README.md

```diff
@@ -1062,9 +1074,9 @@ License and Author
     Licensed under the Apache License, Version 2.0 (the "License");
     you may not use this file except in compliance with the License.
     You may obtain a copy of the License at
-    
+
         http://www.apache.org/licenses/LICENSE-2.0
-    
+
```

Comment:
Please, do not remove these lines with spaces in the license section. They may seem accidental, but they are there to prevent some buggy markdown viewers from breaking the box that contains the license.